Cargo.toml

```
@@ -39,7 +39,7 @@ let_underscore_must_use = "allow"
 
 
 [dependencies]
-zksync-ethers-rs = { git = "https://github.com/lambdaclass/zksync-web3-rs", branch = "zksync-ethers-rs-v1" }
+zksync-ethers-rs = { git = "https://github.com/lambdaclass/zksync-web3-rs", branch = "fd-get-l2-token-addr" }
```

The PR was already merged into `zksync-ethers-rs-v1`

```suggestion
zksync-ethers-rs = { git = "https://github.com/lambdaclass/zksync-web3-rs", branch = "fd-get-l2-token-addr" }
```